compiler/damlc/tests/daml-test-files/ProposalDesugared.daml

```diff
@@ -3,7 +3,8 @@
 
 -- This test demonstrates how the desugared form of the generical proposal
 -- workflow and its instantiation for an IOU look like.
--- @SINCE-LF 1.5
+-- Limited to 1.dev since we do not expose toAnyTemplate/fromAnyTemplate for older LF versions
+-- @SINCE-LF 1.dev
```

Comment:
```suggestion
-- @SINCE-LF 1.7
```
Otherwise we might forget to test it for DAML-LF 1.7 after we froze it. If we decide to not roll this into 1.7 the test will fail, but that's fine.